app/templates/resources/add.html

```diff
@@ -0,0 +1,133 @@
+{% extends 'layouts/base.html' %}
+{% import 'macros/form_macros.html' as f %}
+
+{% block content %}
+    <div class="ui grid container">
+        <div class="eight wide computer sixteen wide mobile centered column">
+            <h2 class="ui dividing header">Add a resource</h2>
+
+            {% set flashes = {
+                'error':   get_flashed_messages(category_filter=['form-error']),
+                'warning': get_flashed_messages(category_filter=['form-check-email']),
+                'info':    get_flashed_messages(category_filter=['form-info']),
+                'success': get_flashed_messages(category_filter=['form-success'])
+            } %}
+
+            {{ f.begin_form(form, flashes) }}
+
+            <div class="two fields">
+                {{ f.render_form_field(form.name) }}
+                {{ f.render_form_field(form.website) }}
+            </div>
+
+            {{ f.render_form_field(form.description) }}
+
+            {{ f.render_form_field(form.autocomplete) }}
+
+            <div class="two fields">
+                {{ f.render_form_field(form.street_number) }}
+                {{ f.render_form_field(form.route) }}
+            </div>
+
+            {{ f.render_form_field(form.locality) }}
+
+            <div class="two fields">
+                {{ f.render_form_field(form.administrative_area_level_1) }}
+                {{ f.render_form_field(form.postal_code) }}
+            </div>
+
+            {{ f.form_message(flashes['error'], header='Something went wrong.', class='error') }}
+            {{ f.form_message(flashes['warning'], header='Check your email.', class='warning') }}
```

Comment:
@aharelick not really sure what these flashes are doing honestly, do we need any of them?